addOns/encoder/src/main/java/org/zaproxy/addon/encoder/EncodeDecodeParamPanel.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2012 The ZAP Development Team
```

Comment:
I left this dated 2012. Since the only thing that changed was the key names.

I did re-date the Param class itself as it was essentially re-written.